### ISSUE TITLE
FTBS 6.5.0 Kernel Fedora 39 Rawhide

### ISSUE THREAD

**Issue by knight-of-ni** [MythTV/mythtv#778]
* **Platform**: Fedora 39 Rawhide

* **MythTV version**: latest fixes/33 branch

* **Package version**: [mythtv-33.1-2.14.20230714gitbeaf2bacbe.fc39](https://koji.rpmfusion.org/koji/buildinfo?buildID=26077)

* **Component**: libmythbase

### What steps will reproduce the bug?

build latest fixes/33 branch against 6.5.0 kernel

### How often does it reproduce? Is there a required condition?
always

### What is the expected behaviour?

builds normally

### What do you see instead?

```
make[2]: Entering directory '/builddir/build/BUILD/mythtv-33.1/mythtv/libs/libmythbase'
g++ -c -pipe -D_FILE_OFFSET_BITS=64 -DPIC -O2 -fexceptions -g -grecord-gcc-switches -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -std=c++17 -faligned-new -fPIC -DQT_DISABLE_DEPRECATED_BEFORE=0x050c00 -msse -pthread -I/usr/include/soundtouch -Wextra -Wpointer-arith -Wundef -fvisibility-inlines-hidden -Wdouble-promotion -Wduplicated-cond -Wlogical-op -Wmissing-declarations -Wnull-dereference -Woverloaded-virtual -Wshadow -funit-at-a-time -Wzero-as-null-pointer-constant -Wno-complain-wrong-lang -fomit-frame-pointer -fno-devirtualize -fvisibility=hidden -std=gnu++1z -D_REENTRANT -fPIC -DMMX -Dusing_libcec -D_GNU_SOURCE -DUSING_DBUS -DRUNPREFIX=\"/usr\" -DLIBDIRNAME=\"lib64\" -DMBASE_API -DUSING_LIBDNS_SD -DUSING_X11 -DHAVE_LIBUDFREAD -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_XML_LIB -DQT_DBUS_LIB -DQT_SQL_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -I/usr/include/qt5 -I/usr/include/libxml2 -I/usr/include/X11 -I../../external/qjsonwrapper -I.. -I/usr/include/qt5/QtWidgets -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtXml -I/usr/include/qt5/QtDBus -I/usr/include/qt5/QtSql -I/usr/include/qt5/QtNetwork -I/usr/include/qt5/QtCore -Imoc -I/../lib64/qt5/mkspecs/linux-g++ -o obj/mythmiscutil.o mythmiscutil.cpp
In file included from mythmiscutil.cpp:706:
/usr/include/linux/ioprio.h:110:43: error: expected primary-expression before 'int'
  110 | static __always_inline __u16 ioprio_value(int class, int level, int hint)
      |                                           ^~~
/usr/include/linux/ioprio.h:110:54: error: expected primary-expression before 'int'
  110 | static __always_inline __u16 ioprio_value(int class, int level, int hint)
      |                                                      ^~~
/usr/include/linux/ioprio.h:110:65: error: expected primary-expression before 'int'
  110 | static __always_inline __u16 ioprio_value(int class, int level, int hint)
      |                                                                 ^~~
/usr/include/linux/ioprio.h:110:73: error: expression list treated as compound expression in initializer [-fpermissive]
  110 | static __always_inline __u16 ioprio_value(int class, int level, int hint)
      |                                                                         ^
mythmiscutil.cpp: In function 'bool myth_ioprio(int)':
mythmiscutil.cpp:727:22: error: 'ioprio_value' cannot be used as a function
  727 |     int new_ioprio = IOPRIO_PRIO_VALUE(new_ioclass, new_iodata);
      |                      ^~~~~~~~~~~~~~~~~
mythmiscutil.cpp:739:22: error: 'ioprio_value' cannot be used as a function
  739 |         new_ioprio = IOPRIO_PRIO_VALUE(IOPRIO_CLASS_BE, new_iodata);
      |                      ^~~~~~~~~~~~~~~~~
make[2]: *** [Makefile:1690: obj/mythmiscutil.o] Error 1
make[2]: Leaving directory '/builddir/build/BUILD/mythtv-33.1/mythtv/libs/libmythbase'
```

### Additional information

TLDR: This might be fixable by simply merging commit 7d6c7d6 into fixes/33 branch.

**Comment by knight-of-ni** [comment]
That did it. Thanks!
